src/fuzz_tests.zig

```diff
@@ -0,0 +1,55 @@
+const std = @import("std");
+const assert = std.debug.assert;
+
+const flags = @import("./flags.zig");
+const fuzz = @import("./testing/fuzz.zig");
+const fatal = flags.fatal;
+
+const log = std.log.scoped(.fuzz);
+
+// NB: this changes values in `constants.zig`!
+pub const tigerbeetle_config = @import("config.zig").configs.test_min;
+
+const Fuzzers = .{
```

Comment:
Looks like the forest fuzz is missing here? (and `manifest_level_fuzz.zig` but IIRC that was broken? Perhaps best to include it as a comment...)

Or maybe we can build this list from searching for `_fuzz.zig` or still keep it static but assert its equal? (of course that's easier when all fuzzers build!)

Reply:
:fearful: `manifest_level_fuzz` doesn't compile, and `forest_fuzz` crashes with null pointer! Will deal with that in a separate PR. 

I don't think we need to specifically protect from missing fuzzers --- given that this is now the single entry point, I think this mistake will be unlikely for any new fuzzers. 

What would be cool though, is to check that any `.zig` file we have is used by _something_, not sure how to do that.

Reply:
>What would be cool though, is to check that any .zig file we have is used by something, 

https://github.com/tigerbeetle/tigerbeetle/pull/1404